.prettierrc

```diff
@@ -1,13 +1,15 @@
 {
-  "printWidth": 140,
-  "singleQuote": true,
-  "tabWidth": 4,
-  "useTabs": false,
-  "arrowParens": "avoid",
-  "overrides": [{
-    "files": ".github/**/*.yml",
-    "options": {
-      "tabWidth": 2
-    }
-  }]
+    "printWidth": 140,
+    "singleQuote": true,
+    "tabWidth": 4,
```

Comment:
Does this set the spaces to 4 instead of 2?

Reply:
Yes, this is the blueprint old configuration.
I didn't change.
https://github.com/jhipster/generator-jhipster-quarkus/blob/2abf8d83b979f58352ead1096d973c9031c94a66/.prettierrc#L4

Reply:
It's up to the maintainer to decide indentation.

Reply:
It's quite easy to change. Change this file and run `npx prettier . --write`.